domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQTuple.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017 the original author or authors.
+ *
+ * This file is part of jBB Application Project.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.frontend.api.faq.model;
+
+/**
+ * Created by Piotr on 2017-06-07.
+ */
+public interface AcpFAQTuple {
+
+    AcpFAQCategory getFAQCategory();
```

Comment:
FaqCategory should be owner of FaqTuples. I mean, FaqCategory should have list of tuples for me